pypeit/wavecalib.py

```diff
@@ -372,6 +374,7 @@ def __init__(self, msarc, slits, spectrograph, par, lamps, binspectral=None, det
         self.spectrograph = spectrograph
         self.par = par
         self.lamps = lamps
+        self.meta_dict=meta_dict
```

Comment:
Can we put the items in `meta_dict` into `par`?

Reply:
Yes see my point above. We should discuss this. I'm not exactly sure how to proceed here, but I sort of prefer parameters in par and core attributes like decker, grating, etc. in the fitstable.